commit 93ebe15eb77c5dff3fcb42bc2b1967216f89c154
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Tue Mar 14 14:23:06 2017 +0000

    bcm2835-gpio-exp: Copy/paste error adding base twice
    
    brcmexp_gpio_set was adding gpio->gc.base to the offset
    twice, so passing an invalid number to the mailbox service.
    The firmware treated it modulo-8 anyway, but was logging an
    assert every time.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/gpio/gpio-bcm-exp.c b/drivers/gpio/gpio-bcm-exp.c
index 681a91492d4c..d68adafaee4a 100644
--- a/drivers/gpio/gpio-bcm-exp.c
+++ b/drivers/gpio/gpio-bcm-exp.c
@@ -165,8 +165,6 @@ static void brcmexp_gpio_set(struct gpio_chip *gc, unsigned int off, int val)
 
 	gpio = container_of(gc, struct brcmexp_gpio, gc);
 
-	off += gpio->gc.base;
-
 	set.gpio = off + gpio->gc.base;	/* GPIO to update */
 	set.state = val;	/* Output state */
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1117_93ebe15eb77c5dff3fcb42bc2b1967216f89c154.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1117_93ebe15eb77c5dff3fcb42bc2b1967216f89c154.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
